test/data/index.js

```diff
@@ -45,7 +45,11 @@ const testData = Object.assign( // eslint-disable-line prefer-object-spread
   Users
 );
 
-testData.seed = seed;
```

Comment:
Given that `testData.reset()` now automatically reseeds, I don't think we need to expose `seed()` anymore. We could add it back in the future if we need to, but I think it makes things clearer to keep it encapsulated for now.